scripts/modules/b2b-account/quotes.js

```diff
@@ -75,16 +80,79 @@ define([
                     });
                 }
             }
+            if (!self.model.get("b2bAccounts")) {
+                var b2bAccount = new B2BAccountModels.b2bAccounts({ pageSize: 200 });
+                b2bAccount.apiGet().then(function (accounts) {
+                    self.model.set("b2bAccounts", accounts);
+                    self.render();
+                });
+            }
+            
+            $('[data-mz-action="applyfilter"]').on('keyup input', function(e) {
+                e.preventDefault();
+                clearTimeout(timeout);
+                FILTERSTRING = "";
+                var dateValue ="";
+                var nameValue = $(this).val();
+                if ($("#expirationdate").val()!=="")
+                {                    
+                    dateValue  = $("#expirationdate").val();
+                }
+                timeout = setTimeout(function () {
+                    self.filterGrid(nameValue, dateValue, collection);
+                }, 1000);
```

Comment:
This should be 400 ms and should be defined in a constant variable.

Reply:
Done put it as 400 ms but not created a separate variable as it is easy to get the promise code this makes it readable and understandable

Reply:
done